core/processor/abci.go

```diff
@@ -169,6 +170,8 @@ type App struct {
 
 	nilPow  bool
 	nilSpam bool
+
+	maxBatchSize uint64
```

Comment:
I think we need a lock on this incase the netparam is being updated while a incoming batch transaction is being length checked.

Reply:
fair enough. will do.

Reply:
addressed